res_syntax/tests/printer/expr/UncurriedByDefault.res

```diff
@@ -71,6 +71,14 @@ type callback4 = (. ReactEvent.Mouse.t) => unit as 'callback
 type callback5 = (. ReactEvent.Mouse.t) => (unit as 'u)
 type callback6 = ((. ReactEvent.Mouse.t) => unit) as 'callback
 
+let foo = (. ()) => ()
+let fn = (_x): ((. unit) => unit) => foo
+let fooC = () => ()
+let fnC = (_x): ((unit) => unit) => fooC
+
+let a = ((. ()) => "foo")->Ok
+let aC = (() => "foo")->Ok
+
 @@uncurried.swap
```

Comment:
I'd add
```
let fooU = () => ()
let fnU = (_x): ((unit) => unit) => fooC
let aU = (() => "foo")->Ok
```
after `@@uncurried.swap`.

With expected:
```
let fooU = () => ()
let fnU = (_x): (unit => unit) => fooU
let aU = (() => "foo")->Ok
```

Reply:
Are they broken? Or is this just an additional test?

Reply:
added

Reply:
> Are they broken? Or is this just an additional test?

It didn't work before your fix